packages/api/tests/test-endpoints-granules.js

```diff
@@ -325,3 +329,105 @@ test('move a granule', async (t) => {
     });
   });
 });
+
+test('move a file and update metadata', async (t) => {
+  const bucket = process.env.internal;
+  process.env.bucket = bucket;
+  const buckets = {
+    protected: {
+      name: process.env.internal,
+      type: 'protected'
+    },
+    public: {
+      name: randomString(),
+      type: 'public'
+    }
+  };
+
+  process.env.distEndpoint = 'http://example.com/';
+  await aws.s3().putObject({ Bucket: bucket, Key: `${process.env.stackName}/workflows/buckets.json`, Body: JSON.stringify(buckets) }).promise();
+
+  await aws.s3().createBucket({ Bucket: buckets.public.name }).promise();
+  const newGranule = fakeGranuleFactory();
+  const metadata = fs.createReadStream('tests/data/meta.xml');
+
+  newGranule.files = [
+    {
+      bucket,
+      name: `${newGranule.granuleId}.txt`,
+      filepath: `${process.env.stackName}/original_filepath/${newGranule.granuleId}.txt`
+    },
+    {
+      bucket: buckets.public.name,
+      name: `${newGranule.granuleId}.cmr.xml`,
+      filepath: `${process.env.stackName}/original_filepath/${newGranule.granuleId}.cmr.xml`,
+      filename: `s3://${buckets.public.name}/${process.env.stackName}/original_filepath/${newGranule.granuleId}.cmr.xml`
+    }
+  ];
+
+  await g.create(newGranule);
+
+  await Promise.all(newGranule.files.map(async (file) => {
```

Comment:
`async (file)` is redundant. You are not using `await` in the body of the function so you can just remove `async`